pomfiles/bcjmail-jdk15on.pom

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.bouncycastle</groupId>
```

Comment:
These POM files were downloaded from https://repo1.maven.org/maven2/org/bouncycastle/ for these components